modals/ideas.js

```diff
@@ -0,0 +1,16 @@
+const { getIdeas, setIdeas } = require('./ideas.js');
+
+function updateIdea(ideaId, idea) {
+    let ideas = getIdeas();
+
+    const idea = ideas.find(idea => ideaId === idea.id);
+    const updateIdea = { ...idea };
+
+    const updateIdeas = ideas.reduse((updateIdeas, idea) => {
```

Comment:
Typo

Reply:
Updated the whole function

Reply:
Regarding branch name, I don't think that it's possible to edit the name? "creaste"...

Reply:
No, but you can create a new branch